data/json/mapgen/cosmics_additions/LANCenter.json

```diff
@@ -17,9 +17,13 @@
         "s": "f_null",
         "v": "f_vending_c",
         "w": "f_null",
+        "Y": "f_rack_coat",
         "{": "f_fridge",
         "|": "f_null"
       },
+      "items": {
+        "Y": { "item": "coat_rack", "chance": 40, "repeat": 2 }
+      },
```

Comment:
This line (and most others) is short enough to be condensed to 

```
"items": { "Y": { "item": "coat_rack", "chance": 40, "repeat": 2 } }

```

Web-linter helps with this kind of issues.